src/main/java/bio/terra/service/duos/DuosService.java

```diff
@@ -155,7 +155,7 @@ private void updateLastSynced(DuosFirecloudGroupModel firecloudGroup, Instant la
     UUID id = firecloudGroup.getId();
     try {
       duosDao.updateFirecloudGroupLastSyncedDate(id, lastSyncedDate);
-    } catch (CannotSerializeTransactionException ex) {
+    } catch (PessimisticLockingFailureException ex) {
```

Comment:
Can I ask about this change, was it needed for test migration?

Reply:
No, this was due to a deprecation warning for `CannotSerializeTransactionException`. This was deprecated and the recommendation was to replace it with a more general exception, `PessimisticLockingFailureException`. This exception is a supertype of `CannotSerializeTransactionException` so the code behavior *should* be the same.